code/frameworks/svelte-vite/package.json

```diff
@@ -51,14 +51,15 @@
     "@storybook/svelte": "workspace:*",
     "magic-string": "^0.30.0",
     "svelte-preprocess": "^5.1.1",
+    "svelte2tsx": "^0.7.13",
     "sveltedoc-parser": "^4.2.1",
-    "ts-dedent": "^2.2.0"
+    "ts-dedent": "^2.2.0",
+    "typescript": "^5.3.2"
```

Comment:
My current implementation heavily relies on the TypeScript engine, but it might be possible to achieve the same functionality using `svelte2tsx`’s internal components, such as https://github.com/sveltejs/language-tools/blob/master/packages/svelte2tsx/src/svelte2tsx/nodes/ExportedNames.ts

Reply:
>  it might be possible to achieve the same functionality using only svelte2tsx

Unfortunately, svelte2tsx’s exportedNames only contains export let style props, not rune style props.

Reply:
I'm considering using `@babel/parser`. It’s probably the method used by react-docgen.

Reply:
I switched to using `@babel/parser` and removed the `typescript` dependency.